vpr/src/base/noc_link.cpp

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Suggest making this doxygen compatible. Then you can make a NoC subcategory in the VPR API documentation on the VTR website.

Reply:
fixed